src/lib/wallet.js

```diff
@@ -38,9 +38,30 @@ export default async function initSdk() {
 
   store.commit('setNodeStatus', NODE_STATUS_CONNECTING);
   try {
+    const { getMiddleware, initMiddleware } = useMiddleware({ store });
+    store.watch(
+      (state, getters) => getters.activeNetwork,
+      async (network, oldNetwork) => {
```

Comment:
I am not sure about vue standards but naming variables `new` and `old` always seems ideal.

Reply:
It can't. It is using the variable inside this particular file.